.ci.yaml

```diff
@@ -66,7 +66,7 @@ platform_properties:
       os: Mac-10.15
       device_type: none
       mac_model: Macmini8,1
-      xcode: 12c33
+      xcode: 13a233
```

Comment:
OS will also need to get changed to 12.0.1

Reply:
Per @jmagman I changed all the mac builders to be at Mac-12, rather then pegged at a more specific version.